docs/terraform.rst

```diff
@@ -15,11 +15,31 @@ AWS resources like Cognito user pools.
     Please step cautiously and be careful when using them!
 
 
+Configurations
+==============
+
+All ``terraform`` commands below expect to be run from within a directory
+containing a Terraform configuration (i.e. a set of one or more ``*.tf`` or
+``*.tf.json`` files).
+
+We have the following configurations:
+
+  - :file:`env/production`
+
+To choose which configuration you're working with, you can either:
+
+  1. ``cd`` into the configuration directory before running any ``terraform``
+     command, or
+
+  2. run all ``terraform`` commands with the ``-chdir=<dir>`` option, e.g.
+     ``terraform -chdir=env/production plan``.
+
+
 Setup
 =====
 
-`Install Terraform <https://www.terraform.io/downloads>`__ and then, from the
-root of this repository, run::
+`Install Terraform <https://www.terraform.io/downloads>`__ and then, from a
+configuration directory (e.g. :file:`env/production`), run::
```

Comment:
Suggestion: Make all the examples reference `env/testing`; that seems safer to me.

Reply:
Good suggestion!

Reply:
https://github.com/nextstrain/nextstrain.org/pull/661